cli/dcoscli/node/main.py

```diff
@@ -90,3 +100,50 @@ def _list(json_):
             emitter.publish(output)
         else:
             emitter.publish(errors.DefaultError('No slaves found.'))
+
+
+def _log(follow, lines, master, slave):
+    """ Prints the contents of master and slave logs.
+
+    :param follow: same as unix tail's -f
+    :type follow: bool
+    :param lines: number of lines to print
+    :type lines: int
+    :param master: whether to print the master log
+    :type master: bool
+    :param slave: the slave ID to print
+    :type slave: str | None
+    :returns: process return code
+    :rtype: int
+    """
+
+    if not (master or slave):
+        raise DCOSException('You must choose one of --master or --slave.')
+    lines = int(lines)
```

Comment:
This throws a ValueError exception. We have a help method that shows a nicer error: https://github.com/mesosphere/dcos-cli/blob/master/dcos/util.py#L366-L383

On a similar note, can we add tests for all the failure cases like this one?